code/datums/perks/fate.dm

```diff
@@ -152,6 +169,7 @@
 	W.AddComponent(/datum/component/inspiration, final_oddity, get_oddity_perk())
 	W.AddComponent(/datum/component/atom_sanity, 1, "") //sanity gain by area
 	W.sanity_damage -= 1 //damage by view
+	W.price_tag += rand(2500,5000)
```

Comment:
Loadsamoney

Reply:
Yeah that's a lot of money, maybe (500, 1500) is enough

Reply:
hmm, how about 1000, 2500?

Reply:
reduced to above.